queries/slippage/subquery_batchwise_internal_transfers.sql

```diff
@@ -144,6 +144,11 @@ potential_buffer_trades as (
            solver_name,
            symbol,
            token,
+           sum( case 
+                    when (transfer_type = 'OUT_AMM' or transfer_type = 'IN_AMM') 
+                    then 1 
+                    else 0
+                end) as count_amm_interations,
```

Comment:
We shouldn't need to count these here because we have the number of `dex_swaps` in the batches view. If this works, then I guess its alright, just wanted to let you know we already have this info.